src/components/accessories/laboratory/EditLaboratoryContent.tsx

```diff
@@ -88,12 +90,18 @@ export const EditLaboratoryContent: FC = () => {
         </div>
       </div>
       {open && (
-        <ExamForm
-          fields={formFields}
-          handleReset={handleReset}
-          creationMode={creationMode}
-          labWithRowsToEdit={labWithRows ?? {}}
-        />
+        <PermissionWrapper
+          permissions={
+            creationMode ? ["exam.create"] : ["exam.read", "exam.update"]
```

Comment:
Do we need the read permission in order to show the examform in edit mode?

Reply:
BEside, if we already start with the usePermission hook I think that would be great to keep the same pattern as long as possibile inside the component.